src/components/Trade/services/useDefaultAssetsService.ts

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Would still make this live under `./hooks`, as it is a service but first and foremost a hook

Reply:
Hmm, does it then follow that _all_ services are hooks as they use state?

If we want to delineate only by the `Service` suffix instead of using folder structure that's cool too, but I'll move them all in another PR.

Reply:
Yeah, definitely something we can and should tackle in a follow-up PR!

The way I see it is if we have a service as a regular function, it's totally fine to namespace it under `./services` but anything that's a hook should live under `./hooks`, regardless of *what* the hook is actually doing.
I guess we could get more granular and namespace them under `./hooks/services` if we really want to make it clear that these are not just your regular utility hooks, but rather service hooks, however I'm not sure whether or not it's worth the mental gymnastics of thinking "what is a service?"